shared/references/cce-sle12-avail.txt

```diff
@@ -1,7 +1,9 @@
-CCE-92282-3
-CCE-92283-1
-CCE-92284-9
-CCE-92285-6
+CCE-92276-5
+CCE-92277-3
+CCE-92278-1
+CCE-92279-9
+CCE-92280-7
+CCE-92281-5
```

Comment:
Why are you including new CCEs in this list?

Reply:
Hello @marcusburghardt 
I will check it out. 
Rumen